documentation/Apple Unified Logging and Activity Tracing formats.asciidoc

```diff
@@ -330,13 +328,29 @@ The Catalog process information sub system is 6 bytes of size and consists of:
 [cols="1,1,1,5",options="header"]
 |===
 | Offset | Size | Value | Description
-| 0 | 2 | | [yellow-background]*Unknown (identifier)*
-| 2 | 2 | | [yellow-background]*Unknown (sub system)*
-| 4 | 2 | | [yellow-background]*Unknown (category)*
+| 0 | 2 | | Identifier
+| 2 | 2 | | Sub system offset (into subsystem strings)
```

Comment:
What is this offset relative to?

Reply:
It is a byte offset into 'Catalog sub system strings'. You can say it's relative to that.